oauthproxy_test.go

```diff
@@ -563,20 +569,22 @@ func TestBasicAuthWithEmail(t *testing.T) {
 		rw := httptest.NewRecorder()
 		req, _ := http.NewRequest("GET", opts.ProxyPrefix+"/testCase1", nil)
 
-		proxy := NewOAuthProxy(opts, func(email string) bool {
+		proxy, err := NewOAuthProxy(opts, func(email string) bool {
 			return email == emailAddress
 		})
+		assert.Equal(t, nil, err)
```

Comment:
Is there a benefit to this over `assert.NoError(t, err)` or just a style preference?

Reply:
Not really, in fact `NoError` is probably better, have changed a bunch